src/SocialAuthDataHandler.php

```diff
@@ -1,69 +1,20 @@
 <?php
 
 namespace Drupal\social_auth;
-
+use Drupal\social_api\SocialApiDataHandler;
 use Symfony\Component\HttpFoundation\Session\SessionInterface;
 
 /**
  * Variables are written to and read from session via this class.
  */
-class SocialAuthDataHandler {
-  protected $session;
-  protected $sessionPrefix;
-
+class SocialAuthDataHandler extends SocialApiDataHandler {
   /**
    * Constructor.
    *
    * @param \Symfony\Component\HttpFoundation\Session\SessionInterface $session
    *   Used for reading data from and writing data to session.
    */
   public function __construct(SessionInterface $session) {
```

Comment:
We don't need to define anything in the class body. Only extending SocialApiDataHandler should be enough